packages/sample/hello/hello.go

```diff
@@ -1,11 +1,28 @@
 package main
 
-func Main(args map[string]interface{}) map[string]interface{} {
-	name, ok := args["name"].(string)
-	if !ok {
-		name = "stranger"
+import (
+	"errors"
+	"fmt"
+)
+
+type Request struct {
+	Location string `json:"location"`
+}
+
+type Response struct {
+	StatusCode int
+	Headers    map[string]string
+	Body       string `json:"body"`
+}
+
+func Main(in Request) (*Response, error) {
+	if in.Location == "" {
+		return nil, errors.New("location must be passed")
 	}
-	msg := make(map[string]interface{})
-	msg["body"] = "Hello " + name + "!"
-	return msg
+
+	loc := fmt.Sprintf("Hello, from %s", in.Location)
+
+	return &Response{
+		Body: loc,
+	}, nil
```

Comment:
```suggestion
	return &Response{
		Body: fmt.Sprintf("Hello, from %s", in.Location),
	}, nil
```

Reply:
The existing format may be more convenient for someone who’s going to start changing this code sample.

Reply:
fair!

Reply:
its a  small tweak, i originally did it that way in my head because it felt  like it divided it into more logical steps. I think regardless its a  minor change that someone interest can still hop into.